internal/core/item_rendering.rs

```diff
@@ -286,6 +287,17 @@ pub trait RenderBorderRectangle {
     fn border_color(self: Pin<&Self>) -> Brush;
 }
 
+/// Trait for an item that represent an Image of the renderer
```

Comment:
I understand this being _input_ to the renderer, not something coming from the renderer.

```suggestion
/// Trait for an item that represents an Image towards the renderer
```